cores/esp8266/core_esp8266_si2c.cpp

```diff
@@ -161,65 +160,25 @@ static Twi twi;
 
 void Twi::setClock(unsigned int freq)
 {
+    if (freq < 250)  // minimum freq 250Hz to prevent overflow @ 232Hz
```

Comment:
As discussed, this probably should be limited to 1KHz.

Reply:
I'm good with 1Khz, tested to 250Hz just in case.  Below that at some point the slave will timeout

Reply:
Here's the commit with the change:
https://github.com/esp8266/Arduino/pull/6934/commits/9cf109546fc0d198c5d91005745dde80d2d8b611
Travis running with it now.